src/Symfony/Component/Form/Extension/Csrf/Type/FormTypeCsrfExtension.php

```diff
@@ -73,6 +76,7 @@ public function finishView(FormView $view, FormInterface $form, array $options):
             $csrfForm = $factory->createNamed($options['csrf_field_name'], HiddenType::class, $data, [
                 'block_prefix' => 'csrf_token',
                 'mapped' => false,
+                'attr' => $this->fieldAttr + ['autocomplete' => 'off'],
```

Comment:
The `autocomplete` attribute on `hidden` field seems invalid: https://github.com/symfony/symfony/issues/59294

Reply:
see #59296